config.py

```diff
@@ -11,7 +9,7 @@
 Start by retrieving your API Key from the "Security" section in new Account admin panel to be able to make authenticated requests to the API.
 https://account.gandi.net/
 '''
-api_secret = '---my_secret_API_KEY----'
+api_secret = 'KWyuy0WIBZ0enWAOjGAM9boP'
```

Comment:
This is completely private. I suggest you to regenerate as fast as possible your api key!!!